src/atoms/Button/BaseButton.tsx

```diff
@@ -8,14 +8,22 @@ type HtmlButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement>
 
 type AnchorProps = React.AnchorHTMLAttributes<HTMLAnchorElement>
 
-export type BaseButtonProps =
+export type BaseButtonProps = (
   | (HtmlButtonProps & CommonProps)
   | (AnchorProps & CommonProps)
+) & {
+  visible?: boolean
+}
```

Comment:
Why not adding `visible` to CommonProps?

Reply:
Because it was Friday, I'm not very fast... thank you ☺️